_data/communication.yml

```
@@ -90,15 +90,15 @@ websites:
     url: https://frontapp.com
     img: front_app.png
     tfa:
-      - software
+      - totp
     doc: https://community.frontapp.com/t/q56jf8/activating-two-factor-authentication-2fa
```

```suggestion
    doc: https://help.frontapp.com/t/36249g
```
Shorter URL possible.